cmd/nerdctl/stop.go

```diff
@@ -176,12 +177,12 @@ func stopContainer(ctx context.Context, container containerd.Container, timeout
 		}
 	}
 
-	signal, err := containerd.ParseSignal("SIGKILL")
+	sig, err := signal.ParseSignal("SIGKILL")
```

Comment:
Why?

Reply:
Has been deprecated in favor moby: https://pkg.go.dev/github.com/containerd/containerd#ParseSignal and gave a deprecated error warning.